rasterio/rio/rio.py

```diff
@@ -32,13 +32,17 @@
 @cli.command(short_help="Open a data file and start an interpreter.")
 @file_in_arg
 @click.option(
+    '--ipython/--no-ipython',
+    default=True,
+    help='Use IPython as interpreter.')
+@click.option(
     '-m',
     '--mode',
     type=click.Choice(['r', 'r+']),
     default='r',
     help="File mode (default 'r').")
 @click.pass_context
-def insp(ctx, input, mode):
+def insp(ctx, input, mode, ipython):
     import rasterio.tool
```

Comment:
@kjordahl Please add a short docstring so that this is communicated via ```rio insp --help```.  Maybe something like: 

```
"""Open the input file within the Python interpreter.  

IPython will be used as the default interpreter, if available.
"""
```